R/helpers_model.R

```diff
@@ -85,10 +83,10 @@ h_model_dual_endpoint_rho <- function(use_fixed,
     assert_true(h_test_named_numeric(rho, permutation.of = c("a", "b")))
     comp$priormodel <- h_jags_join_models(
       comp$priormodel,
-      function() {
+      quote({
         kappa ~ dbeta(rhoa, rhob)
         rho <- 2 * kappa - 1
-      }
+      })
```

Comment:
And this is how the interface is used with quotes. Effectively it's just specifying the body of a function without having the added machinery of a function interface wrapped around it. This avoids assumptions about how the function can be used by providing a narrower interface that provides strictly the data you need.

Reply:
This is great! I really like it